examples/platform/silabs/efr32/spi_multiplex.c

```diff
@@ -15,13 +15,17 @@
  *    limitations under the License.
  */
 
-#include "spi_multiplex.h"
+#if (defined(EFR32MG24) && defined(WF200_WIFI))
+#include "sl_wfx.h"
+#endif /* EFR32MG24 && WF200_WIFI */
+
 #include "dmadrv.h"
 #include "em_bus.h"
 #include "em_cmu.h"
 #include "em_gpio.h"
 #include "em_ldma.h"
 #include "em_usart.h"
+#include "spi_multiplex.h"
```

Comment:
should be in the #ifdef above with the related wf200 stuff

Reply:
this was not added additionally, it was just moved from top to bottom by the restyler.

Reply:
humm ok i see but spi_multiplex.h isn't used/needed for none-wifi ncp builds it should probably be in a ifdef still

Reply:
updated